standalone/include/psen_scan_v2_standalone/protocol_layer/scanner_state_machine_def.h

```diff
@@ -146,6 +146,8 @@ inline void ScannerProtocolDef::sendStartRequest(const T& event)
     auto host_ip{ control_client_.hostIp() };
     config_.hostIp(host_ip.to_ulong());
     PSENSCAN_INFO("StateMachine", "No host ip set! Using local ip: {}", host_ip.to_string());
+    PSENSCAN_INFO("StateMachine", "hello from extra", "");
```

Comment:
unwanted change. this should have been cleared by self. not expected to leave such extra unwanted changes.